pvacseq/lib/net_chop.py

```diff
@@ -91,19 +92,24 @@ def main(args_input = sys.argv[1:]):
             score = -1
             pos = 0
             sequence_name = False
+            cleavage_scores = {}
             for line in results[i].split('\n'):
                 data = [word for word in line.strip().split(' ') if len(word)]
                 currentPosition = data[0]
+                isCleavage = data[2]
+                if isCleavage is not 'S':
+                    continue
                 currentScore = float(data[3])
                 if not sequence_name:
                     sequence_name = data[4]
-                if currentScore > score:
-                    score = currentScore
-                    pos = currentPosition
+                cleavage_scores[currentPosition] = currentScore
+            ranked_cleavage_scores = sorted(cleavage_scores.items(), key=lambda x: x[1], reverse=True)
```

Comment:
Do we need the full ordered list by scores?  If not, this could be replaced with a `max()` to find the "Best" one.